vendor/src/github.com/docker/libcontainer

```diff
@@ -0,0 +1 @@
+../../../../
```

Comment:
@dqminh I don't want to execute a docker container to compile nsinit or execute tests. Why is it bad to have this link?